x-pack/plugins/security/public/management/role_mappings/role_mappings_grid/create_role_mapping_button/create_role_mapping_button.tsx

```diff
@@ -21,7 +21,7 @@ interface CreateRoleMappingButtonProps {
 export const CreateRoleMappingButton = ({ history }: CreateRoleMappingButtonProps) => {
   return (
     <EuiButton
-      iconType="plusInCircle"
+      iconType="plusInCircleFilled"
```

Comment:
This is just to make this page consistent with our other pages. It is unrelated to the main changes of the PR.